tests/resources/terraform/terraform_mapping_specific_functions.yaml

```diff
@@ -1,6 +1,7 @@
 trustzones:
-  - id: b61d6911-338d-46a8-9f39-8dcd24abfe91
+  - id:   b61d6911-338d-46a8-9f39-8dcd24abfe91
```

Comment:
I will put on the **id** field **public-cloud** in order not to create confusion about the same UUID on both places